README.md

```diff
@@ -1,41 +1,67 @@
-############################################
 # NF1 Inactivation Classifier for Glioblastoma
-#
-## (C) 2016 Gregory Way, Robert Allaway, Rebecca Barnard, Stephanie Bouley, Brian O'Roak, Camilo Fadul, Yolanda Sanchez, and Casey Greene
-############################################
+
+**(C) 2016 Gregory Way, Robert Allaway, Rebecca Barnard,
```

Comment:
Disagree. I don't think the copyright statement should be a header. According to [some site](http://practicaltypography.com/headings.html): "Headings are signposts for readers that reveal the structure of your argument".

If you want a header you could put `## Copyright`, but I don't think that's essential.